core/html_api.php

```diff
@@ -430,25 +427,9 @@ function require_js( $p_script_path ) {
 	$g_scripts_included[$p_script_path] = $p_script_path;
 }
 
-/**
- * (6a) Javascript...
- * @return void
- */
-function html_head_javascript() {
-	global $g_scripts_included;
-
-	echo "\t" . '<script type="text/javascript" src="' . helper_mantis_url( 'javascript_config.php' ) . '"></script>' . "\n";
-	echo "\t" . '<script type="text/javascript" src="' . helper_mantis_url( 'javascript_translations.php' ) . '"></script>' . "\n";
-	html_javascript_link( 'jquery-1.11.3.min.js' );
-	html_javascript_link( 'jquery-ui-1.11.4.min.js' );
-	html_javascript_link( 'common.js' );
-	foreach ( $g_scripts_included as $t_script_path ) {
-		html_javascript_link( $t_script_path );
-	}
-}
 
 /**
- * (7) End the <head> section
+ * (6) End the <head> section
```

Comment:
I'm not sure these numbers add much value.  I suggest remove them.  We already have a place where we show the order all methods are called.  See line 31.